lib/path.js

```diff
@@ -46,6 +46,14 @@ const {
   validateObject,
   validateString,
 } = require('internal/validators');
+const {
+  toPathIfFileURL: URLToPathIfFileURL,
```

Comment:
Preferably, a variable with camelCasing. I can see its little complex as the existing functions already reserved the names.

However, something like `getPathFromFileURL ` or something similar would be nice to have.